client/index.html

```diff
@@ -15,7 +15,7 @@
 
 	<link rel="stylesheet" href="css/bootstrap.css">
 	<link rel="stylesheet" href="css/style.css">
-	<link id="theme" rel="stylesheet" href="<%= typeof(theme) !== "undefined" ? theme : "themes/example.css" %>">
+	<link id="theme" rel="stylesheet" href="<%= theme %>">
```

Comment:
This is a somewhat revert of https://github.com/thelounge/lounge/pull/474, which was fixed at the time but tested on an outdated master, so it was not needed anymore.